src/test/testRunner.ts

```diff
@@ -31,7 +31,7 @@ async function main(): Promise<void> {
       'src',
       'test',
       'testFixtures',
-      'settings.json',
+      'settings.json'
```

Comment:
Why did you delete this trailing comma? It's unrelated and also it's preferable to have trailing commas in sequences, just like you added to `cliArgs` above.

Reply:
This happened when I saved the file!
Maybe prettier did it ?

Reply:
This would be weird — why would it remove it in some places but add in others?

Reply:
So, basically all the code written by me follows this rule -> the last element in the list does not have a comma. So, the code that is written by someone else maybe does not have this rule (or maybe, the user never uses prettier).
And if I happen to open the code, make a change and save it, my rules are applied to the file.

I think we should have a prettier config file for this project, maybe ?

Reply:
Maybe. The reason that trailing commas are useful is that when you add/remove items, it produces a clean Git diff not touching unrelated lines.

Reply:
Sure. Understood. Will take care of it from next time !